client/src/components/ReviewModal/ReviewModal.scss

```diff
@@ -17,11 +17,35 @@
   }
 }
 
+// The provided modal header has 56px wide close icon which we want to avoid
+.ant-modal-header {
+  padding-right: 56px !important;
+}
+
 .modal-title {
   flex-flow: wrap;
 
-  @media only screen and (max-width: 650px) {
-    .action-items {
+  .modal-buttons {
```

Comment:
sorry for the big diff, the `modal-btn` styles were spread across 2 files unnecessarily, so I merged them in this PR